py/test_parse_and_notify.py

```diff
@@ -2,65 +2,60 @@
 
 import pytest
 
-from parse_and_notify import NewTopic, UpdatedTopic, make_markdown_table
+from parse_and_notify import Destinations, NewTopic, UpdatedTopic, make_table
 
 TABLE_TESTS = [
-    pytest.param([], [], "%%%\n||\n||\n%%%", id="Empty table"),
-    pytest.param(
-        ["col1", "col2"],
-        [],
-        "%%%\n|col1|col2|\n|-|-|\n%%%",
-        id="Table with header",
-    ),
+    pytest.param([], [], "", id="Empty table"),
+    pytest.param(["col1", "col2"], [], "", id="Table with just headers"),
     pytest.param(
         ["col1", "col2"],
         [["val1", "val2"], ["val3", "val4"]],
-        "%%%\n|col1|col2|\n|-|-|\n|val1|val2|\n|val3|val4|\n%%%",
+        "```| col1 | col2 |\n| ---- | ---- |\n| val1 | val2 |\n| val3 | val4 |\n```",  # noqa
         id="Table with header and rows",
     ),
 ]
 
 
 @pytest.mark.parametrize("headers, content, expected", TABLE_TESTS)
-def test_markdown_table(
+def test_table(
     headers: Sequence[str],
     content: Sequence[Sequence[str | int | None]],
     expected: str,
 ) -> None:
-    assert make_markdown_table(headers, content, False, None) == expected
+    assert make_table(headers, content, None, Destinations.SLACK) == expected
```

Comment:
IF you added two different ways this work, you'll have to test both now. This test is meant to test the corner cases. It is not replaced by `test_topicctl` below. That one tests multiple pieces together.